packages/contracts-bedrock/src/L1/L1CrossDomainMessenger.sol

```diff
@@ -53,4 +53,16 @@ contract L1CrossDomainMessenger is CrossDomainMessenger, ISemver {
     function _isUnsafeTarget(address _target) internal view override returns (bool) {
         return _target == address(this) || _target == address(PORTAL);
     }
+
+    /// @inheritdoc CrossDomainMessenger
+    function paused() public view override returns (bool) {
+        return PORTAL.paused();
```

Comment:
PORTAL.paused() is [here](https://github.com/ethereum-optimism/optimism/pull/7310/files#diff-f5f7dc90c748b7d98110fa1cc80f3bedd5aa9f299b691c51b9bb473a20b526caR158), it just calls out to the SuperchainConfig to get the paused status. 

IMO this should be changed to also call to the SystemConfig, but this approach made the refactoring easier so far.

Reply:
Yes, I noticed this open item this morning.